workflows/offline_ml_diags/offline_ml_diags/_plot_input_sensitivity.py

```diff
@@ -78,3 +81,88 @@ def plot_jacobian(model: fv3fit.keras._models.DenseModel, output_dir: str):
         plt.tight_layout()
         with fsspec.open(os.path.join(output_dir, LINE_NAME), "wb") as f:
             fig.savefig(f)
+
+
+def plot_rf_feature_importance(
+    input_feature_indices: Mapping[str, Tuple[int, int]],
+    wrapped_model: SklearnWrapper,
+    output_dir: str,
+) -> None:
+
+    importances = []
+    for member in wrapped_model.model.regressors:
+        importances.append(member.feature_importances_)
+    mean_importances = np.array(importances).mean(axis=0)
+    std_importances = np.array(importances).std(axis=0)
+
+    n_vector_features = len(
+        [
+            var
+            for var in input_feature_indices
+            if ((input_feature_indices[var][1] - input_feature_indices[var][0]) > 1)
+        ]
+    )
+    fig, axs = plt.subplots(
+        1, n_vector_features + 1, figsize=(6 * n_vector_features, 4), squeeze=False,
+    )
+    axs = _subplot_vector_feature_importances(
+        axs, input_feature_indices, mean_importances, std_importances
+    )
+    axs = _subplot_scalar_feature_importances(
+        axs, input_feature_indices, mean_importances, std_importances
+    )
+    for ax in axs[0]:
+        ax.set_ylim(0.0, max(mean_importances) * 1.1)
+    axs[0][0].set_ylabel("feature importance")
+    plt.tight_layout()
+
+    with fsspec.open(os.path.join(output_dir, RF_FEATURE_IMPORTANCE_NAME), "wb") as f:
+        fig.savefig(f)
+
+
+def _subplot_vector_feature_importances(
+    axs, variable_indices, mean_importances, std_importances
+):
+    vector_features = [
+        var
+        for var in variable_indices
+        if (variable_indices[var][1] - variable_indices[var][0] > 1)
+    ]
+    for i, feature in enumerate(vector_features):
+        dim_length = variable_indices[feature][1] - variable_indices[feature][0]
+        axs[0, i].errorbar(
+            range(dim_length),
+            mean_importances[
+                variable_indices[feature][0] : variable_indices[feature][1]
+            ],
+            std_importances[
+                variable_indices[feature][0] : variable_indices[feature][1]
+            ],
+        )
+        axs[0, i].set_xlabel(f"{feature}, model level")
+    return axs
+
+
+def _subplot_scalar_feature_importances(
+    axs, variable_indices, mean_importances, std_importances
+):
+    # Plot the scalar feature importances together in the last axes object
+    scalar_features = [
```

Comment:
In general, I think these two helpers could be simplified to just the plotting with passed-in importances.